Basic-Audio-Transformer/README.md

```diff
@@ -2,29 +2,72 @@ OpenTok.js Basic Audio Transformer
 =======================
 
 This sample application shows how to use a basic transformer with the Vonage
-video APIs. It is very similar to the [Basic Video Chat](../Basic%20Video%20Chat/) example but it adds [transformer.js](./js/transformer.js), [worker-media-processor.js](./js/worker-media-processor.js), and [worker.js](./js/worker.js) files which create an audio transformer, a web worker, and a separate thread for the web worker publishing a transformed audio stream that publishes the audio below the cutoff value of 100Hz. 
+Video APIs. It is very similar to the [Basic Video Chat](../Basic%20Video%20Chat/) example but it adds [transformer.js](./js/transformer.js), [worker-media-processor.js](./js/worker-media-processor.js), and [worker.js](./js/worker.js) files which create an audio transformer, a web worker, and a separate thread for the web worker publishing a transformed audio stream that publishes the audio below the cutoff value of 100Hz.
 
 ## Demo
 
 You can see a demo of this sample running at [opentok.github.io/opentok-web-samples/Basic-Audio-Transformer/](https://opentok.github.io/opentok-web-samples/Basic-Audio-Transformer/)
 
-> **Note** The demo is setup so that a new room is generated based on your public IP address. So will only work if you are connecting from 2 browsers on the same network.
+> **Note** The demo is set up so that a new room is generated based on your public IP address. So it will only work if you are connecting from two browsers on the same network.
```

Comment:
This was copied from `Basic-Video-Chat`. If we need to reword it, we could change to:

```suggestion
It will only work if you are connecting from two browser tabs on the same network.
```